### PR TITLE
Fix typos in ompi/mca subdirectory

### PR DESCRIPTION
**Pull request by luzpaz**
Found via `codespell -q 3 -L bloc,childs,fo,improbe,inout,mmaped,ro`

### CODE REVIEW

**Comment by awlauria** [comment]
@luzpaz can you add the --sign-off

**Comment by awlauria** [comment]
Thanks!

**Comment by awlauria** [comment]
ok to test

File: ompi/mca/mtl/portals4/mtl_portals4_send.c [hunk 1]
```diff
@@ -385,7 +385,7 @@ ompi_mtl_portals4_long_isend(void *start, size_t length, uint32_t contextid, int
     }
 
     /* We have to wait for some GET events.
-       If the first put falls in overflow list, the number of GET event is egal to:
+       If the first put falls in overflow list, the number of GET event is euqal to:
```
Comment (jsquyres) [inline ompi/mca/mtl/portals4/mtl_portals4_send.c:388]:
```suggestion
       If the first put falls in overflow list, the number of GET event is equal to:
```

**Review by jsquyres — COMMENTED** [review]
Just to let you know that someone is actually reading your work, and not just blindly hitting "approve"!  😜

**Comment by jsquyres** [comment]
@luzpaz Sorry, when you use the Github "accept the suggestion" functionality, it adds a new commit that does not have the signoff line, which makes the PR fail CI again. 😦 

Would you mind squashing the 2nd commit into the first?

**Review by jsquyres — APPROVED** [review]

**Pull request merged**